migrations/2_deploy_contracts.js

```diff
@@ -52,11 +57,29 @@ module.exports = function (deployer, network, accounts) {
     .then(() => AtonomiToken.deployed())
     .then((instance) => {
       t = instance
-      if (network === 'web') {
+      if (network === 'web' || network === 'devRepServer') {
         return t.setReleaseAgent(owner, {from: owner})
       }
     })
-    .then((tx) => network === 'web' ? console.log('Owner set as release agent:', tx.receipt.status) : null)
-    .then(() => network === 'web' ? t.releaseTokenTransfer({from: owner}) : null)
-    .then((tx) => network === 'web' ? console.log('Token transfers enabled:', tx.receipt.status) : null)
+    .then((tx) => network === 'web' || network === 'devRepServer' ? console.log('Owner set as release agent:', tx.receipt.status) : null)
+    .then(() => network === 'web' || network === 'devRepServer' ? t.releaseTokenTransfer({from: owner}) : null)
+    .then((tx) => network === 'web' || network === 'devRepServer' ? console.log('Token transfers enabled:', tx.receipt.status) : null)
+
+    // sets up development environment testing for rep server
+    .then(() => network === 'devRepServer' ? a.addNetworkMember(actors.irnNode, false, false, true, '', { from: owner }) : null)
+    .then((tx) => network === 'devRepServer' ? console.log('Network member added (owner):', tx.receipt.status) : null)
+    .then(() => network === 'devRepServer' ? a.addNetworkMember(actors.mfg, false, true, false, 'auditMan', { from: owner }) : null)
+    .then((tx) => network === 'devRepServer' ? console.log('Network member added (mfg): ', tx.receipt.status) : null)
+    .then(() => network === 'devRepServer' ? t.transfer(actors.mfg, 300000000000000000000000000, { from: owner }) : null)
+    .then((tx) => network === 'devRepServer' ? console.log('Tokens transferred from owner to mfg:', tx.receipt.status) : null)
+    // ...registrate, activate
+    .then(() => network === 'devRepServer' ? t.approve(a.address, 300000000000000000000000000, { from: actors.mfg }) : null)
+    .then((tx) => network === 'devRepServer' ? console.log('Mfg approved tokens for Atonomi to withdraw:', tx.receipt.status) : null)
+    .then(() => network === 'devRepServer' ? a.registerAndActivateDevice(deviceId, 'smartphone', devicePublicKey, { from: actors.mfg }) : null)
```

Comment:
You will still need two devices for your testing.  One that is just registered and one that is activated